File: cfgs/pythoncfgs/QUICK_WAY_CHECK_CHARACTERS_STRING.py
Nodes
1;statement;foo_1 = fun_1(foo_2);8;
2;conditional;foo < foo;9;
3;statement;return true;12;
4;conditional;foo_1[foo_2] != foo_3[0];10;
5;statement;return false;11;
6;exit;;
Edges
0;2;
1;3;4;
2;6;
3;2;5;
4;6;
5;

File: cfgs/java_cfgs_foo/QUICK_WAY_CHECK_CHARACTERS_STRING.java
Nodes
1;special;;;
2;statement;int n = fun_1();14;
3;statement;int i = 1;15;
4;conditional;foo_1 < foo_2;16;
5;conditional;fun_1(foo_2) != fun_2(0);17;
6;statement;return false;17;
7;exit;;;
8;statement;i++;17;
9;statement;return true;18;
Edges
0;2;
1;3;
2;4;
3;5;9;
4;6;8;
5;7;
6;
7;4;
8;7;
